### DISCUSSION TITLE
Studio Dashboard - make PRODUCTION environments visibly different than other environments

### DISCUSSION THREAD

**Discussion by rcwestlake** [supabase/supabase#17993] [Feature Requests]
This is a small UI feature but we are maintaining development and production environments. I want to be able to change the color of the header bar or have some clear, visible element to remind me that it's PROD and not DEV. One idea is to have the ability to change the header bar color. 

This is more of a "safety" element.

**Comment by MildTomato** [comment]
How do you handle your environments currently? @rcwestlake 
Sounds like you have 2 hosted projects, one for PROD, one for DEV ?

**Comment by rcwestlake** [comment]
Yep, exactly. Two hosted projects. I then use Github Actions to deploy to the environment based on the branch. Supabase has a guide for it: [local development](https://supabase.com/docs/guides/cli/local-development) and [managing environments](https://supabase.com/docs/guides/cli/managing-environments)

**Reply by MildTomato** [reply]
@rcwestlake did you sign up for Supabase Branching early access?

It won't help very much if you need a long running dev branch, but you can have a development env for every git feat branch then. We are possibly planning on supporting long running env branch later.

Does that sound more helpful, or maybe you want to keep to separate Projects? 
Let us know what you think?

**Reply by rcwestlake** [reply]
Oof... I didn't look at your profile and thought you were just someone getting started with Supabase... not someone who works there haha. You knew about those guides ;). 

I *think* it could be helpful but it will be easiest for me to just have a long-running dev branch. I'm just a solo developer so my answer would probably be different if I was working with a larger team.

**Reply by MildTomato** [reply]
We semi support long running branches; by which I mean you could keep a Preview branch open all the time.
It wouldn't be perfect but it might be a step in the right direction.
We haven't really properly considered actual support for long running branches but perhaps we should.
